overviews/quasiquotes/definition-details.md

```diff
@@ -58,32 +58,32 @@ In deconstruction one can either extract `Modifiers` or annotations, but you can
     scala> val q"@..$annots implicit def f" = q"@foo @bar implicit def f"
     annots: List[universe.Tree] = List(new foo(), new bar())
 
-Considering the fact that definitions might contain various low-level flags added to trees during typechecking it\'s recommended to always extract complete modifiers as otherwise your pattern might not be exhaustive. If you don't care about them just use a wildcard:
+Considering that definitions might contain various low-level flags added to trees during typechecking, it\'s recommended to always extract complete modifiers, or your pattern might not be exhaustive. If you don't care about them just use a wildcard:
 
     scala> val q"$_ def f" = q"@foo @bar implicit def f"
 
 ## Templates
 
-Templates are a common abstraction in definition trees that is used in new expressions, classes, traits, objects, package objects. Although there is no interpolator for it at the moment we can illustrate its structure on the example of new expression (similar handling will apply to all other template-bearing trees):
+Templates are a common abstraction in definition trees that are used in new expressions, classes, traits, objects, and package objects. Although there is no interpolator for it at the moment we can illustrate its structure using the example of a new expression (similar handling will apply to all other template-bearing trees):
 
     q"new { ..$earlydefns } with ..$parents { $self => ..$stats }"
 
-So template consists of:
+The template consists of:
 
-1. Early definitions. A list of val or type definitions. Type definitions are still allowed by they are deprecated and will be removed in the future:
+1. Early definitions. A list of `val` or `type` definitions. Type definitions are still allowed by they are deprecated and will be removed in the future:
 
         scala> val withx = q"new { val x = 1 } with RequiresX"
         withx: universe.Tree = ...
 
         scala> val q"new { ..$earlydefns } with RequiresX" = withx
         earlydefns: List[universe.Tree] = List(val x = 1)
 
-2. List of parents. A list of type identifiers with possibly an optional arguments to the first one in the list:
+2. List of parents. A list of type identifiers with optional arguments to the first one in the list:
```

Comment:
this sentence remains unclear (but if you don't know how to fix it, feel free to just leave it)